stl/inc/variant

```diff
@@ -1600,6 +1625,15 @@ template <class _Ty>
 using _As_variant = // Deduce variant specialization from a derived type
     decltype(_As_variant_(_STD declval<_Ty>()));
 
+template <size_t _Size, class _ListOfIndexVectors, class _Ret, class _Callable, class... _Variants>

```

Comment:
Mild preference to be consistent about `_ListOfIndexVectors` and `_Ret` order. Here _ListOfIndexVectors is first, but _Visit2 has the opposite order.

Reply:
I've made `_Visit_impl` consistent with the other templates that pass `_Ret` before `_ListOfIndexVectors`. (Sorry if I'm stepping on your toes, @NathanSWard, I'm trying to polish everything up to get a clean test run.)

Reply:
No worries!
Thanks for all of the help @CaseyCarter with my PR's 😄